rerun/src/main/java/org/apache/falcon/rerun/handler/RetryHandler.java

```diff
@@ -75,6 +76,9 @@ public void handleRerun(String clusterName, String entityType, String entityName
                         "All retry attempt failed out of configured: "
                                 + attempts + " attempt for entity instance::");
             }
+        } catch (EntityNotRegisteredException ee) {
```

Comment:
Most other listeners other than the ones under rerun seem to just read off the context and not really query the config store for the entity for which notification is generated. So, nothing needs to be fixed immediately.

Agree that we have to have cleaner solution.